index.html

```diff
@@ -8,9 +8,80 @@
     <link rel="stylesheet" href="css/normalize.css">
```

Comment:
Add this line in your `<head>`:
`<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">`
It will ensure that the page is scaled on a mobile. At the moment, everything is tiny 🔍

Reply:
With this, and the use of flexbox on your .content, as suggested, you will find it much easier to position your content on mobile 👍